Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN microdnf clean all && microdnf install -y python3.9 ca-certificates tar gzip
 RUN python3 -m pip install awscli && python3 -m pip install oci-cli && python3 -m pip install rsa --upgrade
 
 
-RUN curl -q -o /usr/local/bin/aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/aws-iam-authenticator && \
+RUN curl -q -o /usr/local/bin/aws-iam-authenticator https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.5.9/aws-iam-authenticator_0.5.9_linux_amd64 && \
```

Comment:
Any particular reason we have taken 0.5.9 version when 0.6.10 is the latest release?

Reply:
This version was showing up as the default on aws website. https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html and all vulnerability scans were getting addressed with it too.